src/memex/views.py

```diff
@@ -82,7 +99,41 @@ def api_config(**settings):
         request_method = ('DELETE', 'GET', 'HEAD', 'POST', 'PUT',)
     settings['request_method'] = request_method + ('OPTIONS',)
 
-    return view_config(**settings)
+    if link_name:
+        link = {'name': link_name,
+                'method': initial_request_method,
+                'route_name': settings.get('route_name'),
+                'description': description,
+                }
+
+        registry = config.registry
+        if not hasattr(registry, 'api_links'):
+            registry.api_links = []
+        registry.api_links.append(link)
+
+    config.add_view(view=view, **settings)
```

Comment:
Does this return something similar to the old `view_config(**settings)`? If yes, we should return it as well, that would make the tests much easier because you don't have to mock things out anymore.